### PR TITLE
Vote fix

### PR DESCRIPTION
**Pull request by mantagen**
- fixes vote bug (wasn't working)
- adds Metropolis as input font
- uses `camelcase` function for bookmark service

### CODE REVIEW

File: server/src/services/search-api.js [hunk 1]
```diff
@@ -91,14 +91,21 @@ const createSearchApiService = ({ config }) => {
...
+    vote
   }) => {
     const userId = user.isDeveloper ? null : user.id;
 
     return axios({
       method: 'PUT',
       url: `${REGSIMPLE_SEARCH_API}/feedback/${snippetId}`,
-      data: { user_id: userId, categories, countries, suggestedCategories },
+      data: {
+        user_id: userId,
```
Comment (Jbarget) [inline server/src/services/search-api.js]: how come this one is snake case not camel?

Comment (mantagen) [reply server/src/services/search-api.js]: not sure ... i just left it unchanged?

Comment (mantagen) [reply server/src/services/search-api.js]: probably causing issues thouhg?

Comment (Jbarget) [reply server/src/services/search-api.js]: will merge and wait for chris's reply to see if we need to change

**Pull request merged**